ocaml/lambda/translobj.ml

```diff
@@ -95,7 +95,7 @@ let transl_label_init_general f =
          let const =
            Lprim (Popaque, [Lconst c], Debuginfo.Scoped_location.Loc_unknown)
          in
-         Llet(Alias, Pgenval, id, const, expr))
+         Llet(Alias, Lambda.structured_constant_layout c, id, const, expr))
```

Comment:
We should put a CR mentioning that we must not have a layout too precise (that's why we added the `Popaque` wrapping). Currently this should end up as `Pgenval` so it should be fine.